routingpy/routers/valhalla.py

```diff
@@ -277,7 +282,29 @@ def parse_direction_json(response, units):
             factor = 0.621371 if units == "mi" else 1
             distance += int(leg["summary"]["length"] * 1000 * factor)
 
-        return Direction(geometry=geometry, duration=int(duration), distance=int(distance), raw=response)
+        origin = response["trip"]["locations"][0]
+        destination = response["trip"]["locations"][-1]
+        departure_time: Optional[datetime] = None
+        arrival_time: Optional[datetime] = None
+        # TODO: maybe we can get rid of that in the future, in case we'll have timezone aware strings in the response:
+        #   https://github.com/valhalla/valhalla/issues/4241
+        if origin.get("date_time"):
+            departure_time = timestamp_to_tz_datetime(
+                origin["date_time"], lonlat_to_timezone(origin["lon"], origin["lat"])
+            )
```

Comment:
valhalla doesn't return the timezone either. it could do that in the `date_time` string [and I think we should](https://github.com/valhalla/valhalla/issues/4241). it won't change any public facing API if we change it in the future, so we can start with this

Reply:
Same as above. I'd take the input timezone.

Reply:
Same comment as above:) destination can have a different timezone